app/views/post/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= provide(:title, "Create post") %>
+<h1>Create Posts</h1>
+
+
+<%= form_for(:post, url: {action: "create"}) do |f| %>
+  <%= f.hidden_field :user_id, value: current_user.id %>
```

Comment:
Why `user_id` need to pass as params? The way I mentioned before for posts controller will solve this. This is not the most convenient way to do that job.